src/Tokenizer.js

```diff
@@ -223,6 +223,15 @@ export class Tokenizer {
           endEarly = true;
         }
 
+        const fencesBeginRegExp = new RegExp(`^( {0,${indent}})(\`\`\`|~~~)`);
```

Comment:
Could you remove the 2nd if statement if this only checked `{0, ${indent - 1}}`?